src/workspaces/features/WorkspaceMembers/WorkspaceMembers.tsx

```diff
@@ -0,0 +1,78 @@
+import Button from "core/components/Button";
+import DataGrid, { BaseColumn } from "core/components/DataGrid";
+import DateColumn from "core/components/DataGrid/DateColumn";
+import { TextColumn } from "core/components/DataGrid/TextColumn";
+
+import { DateTime } from "luxon";
+import { useTranslation } from "react-i18next";
+import { useWorkspacePageQuery } from "workspaces/graphql/queries.generated";
+
+export default function WorkspaceMembers({
+  workspaceId,
+}: {
+  workspaceId: string;
+}) {
+  const { t } = useTranslation();
+  const { data, refetch } = useWorkspacePageQuery({
+    variables: { id: workspaceId, page: 1, perPage: 5 },
+  });
+
+  const onChangePage = ({ page }: { page: number }) => {
+    refetch({
+      page,
+      perPage: 5,
+      id: workspace.id as string,
+    });
+  };
+
+  if (!data?.workspace) {
+    return null;
+  }
+
+  const { workspace } = data;
+
+  return (
+    <DataGrid
```

Comment:
Have you tried the pagination ? It does not look like it works.

Reply:
Yes. I've tried. Can you explain me why you think it doesn't :thinking: ?

Reply:
Do you see the pagination when you set a perPage inferior of the number of members in your workspace ?

Reply:
It shows if I modify the defaultPageSize and set a perPage inferior.
![image](https://user-images.githubusercontent.com/25453621/212114556-d346e34b-f774-4608-83e8-d837d82dbaf2.png)

Reply:
Indeed, my bad. It allowed me to spot an small error in the `defaultPageSize` that was not equal to the perPage argument